php/libraries/NDB_Page.class.inc

```diff
@@ -766,12 +766,12 @@ class NDB_Page implements RequestHandlerInterface
                   $baseurl . '/js/polyfills.js',
                     // $baseurl . '/js/react/react-with-addons' . $min . '.js',
                     // $baseurl . '/js/react/react-dom' . $min . '.js',
-                  $baseurl . ($min === '') ?
+                  $baseurl . ($min === '' ?
                       '/vendor/js/react/react.development.js' :
-                      '/vendor/js/react/react.production.js',
-                  $baseurl . ($min === '') ?
+                      '/vendor/js/react/react.production.js'),
```

Comment:
'/vendor/js/react/react.production'.$min.'.js'), instead of '/vendor/js/react/react.production.js'),